src/index.html

```diff
@@ -3,8 +3,64 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>HTML-CSS-Practice-two</title>
+    <title>HTML-CSS-Practice-1</title>
     <link rel="stylesheet" href="./style/index.css" />
   </head>
-  <body></body>
+  <body>
+    <header class="header-main">
+      <div class="header-content container">
+        <div class="btn-menu">
+          <span class="btn-line"></span>
+          <span class="btn-line"></span>
+          <span class="btn-line"></span>
+        </div>
+
+        <nav class="navbar">
+          <ul class="navbar-list">
+            <li class="navbar-item">
+              <a href="#" class="navbar-link">Çözüm ve Hizmetler</a>
```

Comment:
You can replace the hash # by `javascript:void(0)` to avoid refreshing page when clicking on the link